tests/test_web_ui_options.py

```diff
@@ -0,0 +1,63 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from inmanta.config import Config
+from inmanta.server import config as opt
+from inmanta_ui import config as cfg
```

Comment:
`opt` and `cfg` are chosen very arbitrarily. I would rename then to make it clear what they refer to:
```suggestion
from inmanta.server import config as opt_core
from inmanta_ui import config as opt_ui
```